.github/workflows/build_and_release.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and Release Performous Website
+
+# Controls when the workflow will run
+on:
+  # Run on a schedule to get weekly updates for the Linux containers
+  # and keep the cache fresh for Windows builds. Runs Sundays at midnight.
+  schedule:
```

Comment:
I'm not sure that the schedule is useful for the website. I don't think there will be any changes that sneak in that don't get deployed when master is run :-)

Reply:
True that: fixed